topics/ecology/tutorials/phylogeny-data-prep/tutorial.bib

```diff
@@ -0,0 +1,151 @@
+               RESULTS: We describe the software tools RepbaseSubmitter and
+               Censor, which are designed to facilitate updating and screening
+               the content of Repbase. RepbaseSubmitter is a java-based
+               interface for formatting and annotating Repbase entries. It
+               eliminates many common formatting errors, and automates actions
+               such as calculation of sequence lengths and composition, thus
+               facilitating curation of Repbase sequences. In addition, it has
+               several features for predicting protein coding regions in
+               sequences; searching and including Pubmed references in Repbase
+               entries; and searching the NCBI taxonomy database for correct
+               inclusion of species information and taxonomic position. Censor
+               is a tool to rapidly identify repetitive elements by comparison
+               to known repeats. It uses WU-BLAST for speed and sensitivity,
+               and can conduct DNA-DNA, DNA-protein, or translated
+               DNA-translated DNA searches of genomic sequence. Defragmented
+               output includes a map of repeats present in the query sequence,
+               with the options to report masked query sequence(s), repeat
+               sequences found in the query, and alignments. CONCLUSION: Censor
+               and RepbaseSubmitter are available as both web-based services
+               and downloadable versions. They can be found at
+               http://www.girinst.org/repbase/submission.html
+               (RepbaseSubmitter) and http://www.girinst.org/censor/index.php
+               (Censor).",
+  journal   = "BMC Bioinformatics",
+  publisher = "Springer Science and Business Media LLC",
+  volume    =  7,
+  number    =  1,
+  pages     = "474",
+  month     =  oct,
+  year      =  2006,
+  language  = "en"
+}
+
+@misc{genome-annotation-funannotate,
+author = "Anthony Bretaudeau and Alexandre Cormier and Laura Leroi and Erwan Corre and Stéphanie Robin and Erasmus+ Programme",
+title = "Genome annotation with Funannotate (Galaxy Training Materials)",
+year = "2022",
+month = "03",
+day = "03",
+url = "\url{https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/funannotate/tutorial.html}",
+note = "[Online; accessed Tue May 31 2022]"
+}
+
+@article{Storer2021,
+  doi = {10.1186/s13100-020-00230-y},
+  url = {https://doi.org/10.1186/s13100-020-00230-y},
+  year = {2021},
+  month = jan,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {12},
+  number = {1},
+  author = {Jessica Storer and Robert Hubley and Jeb Rosen and Travis J. Wheeler and Arian F. Smit},
+  title = {The Dfam community resource of transposable element families,  sequence models,  and genome annotations},
+  journal = {Mobile {DNA}}
+}
+
+@misc{genome-annotation-repeatmasker,
```

Comment:
generally you don't need to cite other tutorials in the GTN, you probably can use another way of linking to it that's a bit nicer.